portal-frontend/src/app/features/public/notification/public-notification.component.ts

```diff
@@ -35,6 +37,10 @@ export class PublicNotificationComponent implements OnInit, OnDestroy {
     if (res) {
       const { submission, documents, parcels } = res;
 
+      if (submission.status.code !== NOTIFICATION_STATUS.ALC_RESPONSE) {
+        this.selectedIndex = 0;
+      }
```

Comment:
Nit: Was there a reason why public notification logic is flipped from non-public notification and all other submission types? (`selectedIndex` set to 0 first, then checked for `===`)

Reply:
Notifications are only public if they are currently in the ALC Response status, where as Notifications and Applications become public once they are accepted as having all items. I thought it made sense to reverse the logic to prevent the scrolling.